extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -985,6 +984,8 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
     }
 
     private val mainHandler = Handler(Looper.getMainLooper())
+    /** Registry to control if we need to ignore parsed result for given sourceId. */
+    private val geoJsonRegistry = hashMapOf<String, Boolean>()
```

Comment:
nit: the `geoJsonRegistry` naming doesn't seem very informative

Reply:
Also, is it thread safe? this registry can be accessed by multiple threads at the same time.

Reply:
Changed naming.
It is thread-safe (always called from main thread) - same as was for just `ignoreParsedGeoJson` (it was non-atomic).